benches/hash.rs

```diff
@@ -0,0 +1,57 @@
+use criterion::{black_box, criterion_group, criterion_main, BatchSize, Criterion};
+use miden_crypto::{
+    hash::{Digest, Hasher},
+    ElementHasher, Felt, HashFn,
+};
+use rand_utils::rand_value;
+
+fn rpo256_2to1(c: &mut Criterion) {
+    let v: [Digest; 2] = [Hasher::hash(&[1_u8]), Hasher::hash(&[2_u8])];
+    c.bench_function("RPO256 2-to-1 hashing (cached)", |bench| {
+        bench.iter(|| Hasher::merge(black_box(&v)))
+    });
```

Comment:
I'm not sure what case we are covering here. Do we expect a performance difference based on the input? If yes, this could lead to a more complicated discussion about the constant time property of this operation

Reply:
I think one of the intentions is to verify that there are in fact no performance differences (this was not the case in previous implementations of field arithmetic as it wasn't constant time).